packages/devtools_app/lib/src/screens/memory/panes/allocation_tracing/allocation_profile_tracing_view_controller.dart

```diff
@@ -68,6 +68,9 @@ class AllocationProfileTracingViewController extends DisposableController
   /// The list of classes for the currently selected isolate.
   ValueListenable<List<TracedClass>> get classList => _classList;
   final _classList = ListValueNotifier<TracedClass>([]);
```

Comment:
let's rename to _filteredClassList. This would match naming convention for filtered data in other places